scripts/watermarkstat

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python
```

Comment:
Why change this?

Reply:
Ran into this error in PR tests after #1151 was merged, 
/usr/bin/env: 'python': No such file or directory
https://sonic-jenkins.westus2.cloudapp.azure.com/job/common/job/sonic-utilities-build-pr/2537/console

Had to change the path to fix it

Reply:
That's strange. #1151 shouldn't have had an impact on this. Also, all other executable scripts in the repo still use `#!/usr/bin/env python`. I feel like this change shouldn't be necessary.

Reply:
Not sure how to fix the build failure. seems to be failing every time utilities wheel is built. But if I retain the docker and run the test again using 'python setup.py test', I don't see the issue

Reply:
It sounds like it may be a PATH difference with the sonic-utilities build job in sonic-build-tools.

Reply:
I am not executing the command directly. I am using 'show' commands to invoke that script. The same format is used in many of the utilities tests which are invoking modules under 'scripts' path and some of them also use "/usr/bin/env python". I fail to understand what is special about this particular script